src/Sylius/Bundle/AdminBundle/Resources/views/Product/Show/_variants.html.twig

```diff
@@ -0,0 +1,120 @@
+{% import "@SyliusAdmin/Common/Macro/money.html.twig" as money %}
+
+<h4 class="ui top attached inverted black header">Variants</h4>
+<div class="ui attached segment">
+    <table class="ui celled table">
+        <thead>
+            <tr>
+                <th>Image</th>
+                <th>Name</th>
+                <th>Code</th>
+                <th>Inventory</th>
+                <th>Position</th>
+                <th>Current stock</th>
+                <th>Tax category</th>
+                <th>Pricing</th>
+                <th>Shipping</th>
+                <th>Inventory tracking</th>
+                <th>shipping requirement</th>
+            </tr>
+        </thead>
+        <tbody>
+            {% for variant in product.variants %}
+            <tr>
+                <td>
+                    {% if variant.images.first() == true %}
```

Comment:
Since docs for first() method of Doctrine's Collection are following:

```
/**
     * Sets the internal iterator to the first element in the collection and returns this element.
     *
     * @return mixed
     *
     * @psalm-return T|false
     */
```

I think that we should check whether `{% if variant.images.first() != false %}`